crawl-ref/source/mon-cast.cc

```diff
@@ -4937,6 +4938,80 @@ dice_def resonance_strike_base_damage(const monster &mons)
     return dice_def(3, mons.spell_hd(SPELL_RESONANCE_STRIKE));
 }
 
+static void _sheep_message(int num_sheep, int sleep_pow, actor *foe)
+{
+    string message = "";
+
+    // Determine messaging based on sleep strength.
+    if (sleep_pow >= 125)
+        message = "You are overwhelmed by glittering dream dust!";
+    else if (sleep_pow >= 75)
+        message = "The dream sheep are wreathed in dream dust.";
+    else if (sleep_pow >= 25)
+    {
+        message = make_stringf("The%s dream sheep shake%s wool and sparkle%s.",
+                               num_sheep == 1 ? "" : " herd of",
```

Comment:
Using "herd" here leaves you in an awkward position re singular vs plural verbs. Why not just leave it as "the dream sheep" whether singular or plural, as in the other cases?